odfuzz/config.py

```diff
@@ -9,13 +9,15 @@
     DEFAULT_SAP_CLIENT,
     DEFAULT_URLS_PER_PROPERTY,
     DEFAULT_IGNORE_METADATA_RESTRICTIONS,
+    DEFAULT_HTTP_METHOD_ENABLED,
     ENV_ASYNC_REQUESTS_NUM,
     ENV_DATA_FORMAT,
     ENV_USE_ENCODER,
     ENV_ODFUZZ_CERTIFICATE_PATH,
     ENV_SAP_CLIENT,
     ENV_URLS_PER_PROPERTY,
     ENV_IGNORE_METADATA_RESTRICTIONS,
+    ENV_HTTP_METHOD_ENABLED,
```

Comment:
This was put in to avoid hard-coding "GET" in FuzzerConfig. 
Also, please look into my suggestion on CLI, which would work well with this implementation, and could achieve all HTTP methods in a single run. 
But i'll remove this for now